packages/gatsby-plugin-netlify/src/create-redirects.js

```diff
@@ -21,10 +21,12 @@ export default async function writeRedirectsFile(
       redirectInBrowser, // eslint-disable-line no-unused-vars
       force,
       toPath,
+      statusCode, // eslint-disable-line no-unused-vars
```

Comment:
is the `eslint-disable-line no-unused-vars` needed here?

Reply:
I've removed it.

Reply:
In a very unsurprising development, turns out you were right :+1: